pkg/config/config.go

```diff
@@ -45,23 +47,25 @@ type Config struct {
 func new() *Config {
 	conf := &Config{
 		DevelopmentMode: DEVELOPMENT_MODE, // Do not read this from ENV. See config_development.go to enable.
+		DBBatchSize:     getEnvAsInt("DB_BATCH_SIZE", 500),
 		DBHost:          getEnv("DB_HOST", "localhost"),
+		DBMaxConns:      getEnvAsInt("DB_MAX_CONNS", int(30)),
 		DBPort:          getEnvAsInt("DB_PORT", 5432),
 		DBName:          getEnv("DB_NAME", ""),
 		DBUser:          getEnv("DB_USER", ""),
 		DBPass:          getEnv("DB_PASS", ""),
 		HTTPTimeout:     getEnvAsInt("HTTP_TIMEOUT", 300000), // 5 min
-		MaxConns:        getEnvAsInt("MAX_CONNS", int(30)),
 		KubeConfigPath:  getKubeConfigPath(),
 		PodName:         getEnv("POD_NAME", "local-dev"),
 		PodNamespace:    getEnv("POD_NAMESPACE", "open-cluster-management"),
 		ServerAddress:   getEnv("AGGREGATOR_ADDRESS", ":3010"),
+		SlowLog:         getEnvAsInt("SLOW_LOG", int(1000)), // 1 second
 		Version:         COMPONENT_VERSION,
-		// Use 5min for delete cluster activities and 30 seconds for db reconnect retry
+		// Use 5 min for delete cluster activities and 30 seconds for db reconnect retry
 		MaxBackoffMS: getEnvAsInt("MAX_BACKOFF_MS", 300000), // 5 min
 		// EdgeBuildRateMS:       getEnvAsInt("EDGE_BUILD_RATE_MS", 15000), // 15 sec
 		RediscoverRateMS: getEnvAsInt("REDISCOVER_RATE_MS", 300000), // 5 min
-		// RequestLimit:          getEnvAsInt("REQUEST_LIMIT", 10),
+		RequestLimit:     getEnvAsInt("REQUEST_LIMIT", 50),
```

Comment:
Should we add a comment to explain why we choose 50 as the default for the request limit and same with max db conn?

Reply:
Added comment in code.  
Main reason is to prevent memory from growing and the container getting killed because OOM.  50 is a conservative guess based on what I saw in the scale tests. We can adjust as we learn more.